akka-actor-tests/src/test/scala/akka/util/VersionSpec.scala

```diff
@@ -35,5 +33,12 @@ class ManifestInfoVersionSpec extends AnyWordSpec with Matchers {
       new Version("1.2-foo") should be > new Version("1.2.3")
       new Version("1.2.3-foo") should be > new Version("1.2.3-bar")
     }
+
+    "compare extra without digits" in {
+      new Version("foo") should ===(new Version("foo"))
+      new Version("foo") should !==(new Version("bar"))
+      new Version("foo") should be > new Version("1.2.3")
+      new Version("foo") should be > new Version("bar")
```

Comment:
@raboof I saw your comment

> Looks like that would even work for dynver-like versioning - but I wonder if we should provide a way for users to customize this when they some other particular scheme?

As you see in this test it's possible to have versions without digits. I don't think more flexibility is needed. For the planned features we only need to compare (older/newer) versions and are not interested in major.minor.patch semantics but that might also be useful in the future.

Reply:
Yeah, I think this is a nice versatile default. This seems to be the kind of thing were some organizations will eventually want to follow their own particular conventions, but it seems fine not to open that up just yet.

Reply:
What Ii think is missing in this  PR (when thinking about dynver) is the parsing of the "num of commits since" segment: `1.0.0+3-1234abcd`. Try, for example:

```
      new Version("1.0.10+3-1234abcd") should be < new Version("1.0.11")
```

Reply:
I added support for `+` separator.
However, `+` is a problem for Docker/k8 anyway so it will be replaced `ThisBuild / dynverSeparator := "-"`

Reply:
fixed